pypeit/core/wave.py

```diff
@@ -261,23 +261,23 @@ def flexure_slit():
     return flex_dict
 '''
 
-# TODO I don't see why maskslits is needed in these routine, since if the slits are masked in arms, they won't be extracted
 def flexure_obj(specobjs, maskslits, method, sky_file, mxshft=None):
     """Correct wavelengths for flexure, object by object
 
-    Parameters:
-    ----------
-    method : str
-      'boxcar' -- Recommneded
-      'slitpix' --
-    sky_file: str
+    Args:
```

Comment:
Please add documentation to this code.

Reply:
I'm not adding docs to methods I haven't 
written in this PR.

Reply:
ok done.